roles/os-glance/README.md

```diff
@@ -1,28 +1,50 @@
-# Ansible role: OpenStack-Glance
-
+# os-glance
 This role setups the OpenStack Image service
 
-# Requirements
-You need a previously setup of the OpenStack Identity service
-where this role can create the users, roles, etc.
-
-# Role variables
-All variables of this role are defined in `defaults/main.yml`
-
-## Mandatory variables
-* `glance_user_password`
-* `glance_database_password`
-
-## Handlers
-* `restart glance`: restart the image services
-
-# Dependencies
-This role needs the OpenStack-common (os-common) role.
-
-# Example playbook
-
-# License
+## Requirements
+- A previously setup database server.
+  You can setup one with `mariadb` role
+- A previously setup OpenStack Identity service.
+  You can setup one with `os-keystone` role
+
```

Comment:
[same comment](https://github.com/clearlinux/clear-config-management/pull/20/files#r70143074) applies here for previous 2 lines (point to URL of the roles)